actionpack/lib/action_controller/metal/content_security_policy.rb

```diff
@@ -14,6 +14,29 @@ module ContentSecurityPolicy
     end
 
     module ClassMethods
+      # Overrides parts of the globally confiugured Content-Security-Policy
```

Comment:
```suggestion
      # Overrides parts of the globally configured Content-Security-Policy
```

I'm surprised the spellcheck [didn't catch this](https://github.com/rails/rails/runs/5236692687). :thinking: